public/New-DbaAvailabilityGroup.ps1

```diff
@@ -56,6 +56,14 @@ function New-DbaAvailabilityGroup {
     .PARAMETER Name
         The name of the Availability Group.
 
+    .PARAMETER IsContained
+        Builds the Availability Group as contained. Only supported in SQL Server 2022 or higher.
+
+        https://learn.microsoft.com/en-us/sql/database-engine/availability-groups/windows/contained-availability-groups-overview?view=sql-server-ver16
```

Comment:
I would exclude the "?view=sql-server-ver16" to always point to the latest version.